lambda/matching.ml

```diff
@@ -1664,7 +1667,15 @@ let get_key_constr = function
 
 let get_pat_args_constr p rem =
   match p with
-  | { pat_desc = Tpat_construct (_, _, args) } -> args @ rem
+  | { pat_desc = Tpat_construct (_, {cstr_inlined=Some _}, args) } ->
```

Comment:
I believe this was left over from an earlier treatment of `cstr_arg_layouts` where we had the layouts for each field of an inline record rather than the record itself.  It shouldn't be needed any more, and I've deleted it.